Locker/Helpers/Devices.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Devices.swift
+//  Locker
+//
+//  Created by Zvonimir Medak on 20.10.2021..
+//
+
+import Foundation
+
+class Devices {
+    static let shared = Devices()
+
+    private(set) var devicesURL: URL? = Constants.devicesURL
+
+    private init() {}
+
+    func fetchDevices() {
```

Comment:
```suggestion

    // MARK: - Singleton creation -

    static let shared = Devices()
    private init() {}
    
    // MARK: - Private properties -
    
    private(set) var devicesURL: URL? = Constants.devicesURL

    // MARK: - Public methods -
    func fetchDevices() {
```

Please go through all new files and add marks everywhere they are missing. Other than that, separate code with our coding style guides (public properties -> private properties -> init/lifecycle -> public extension (methods) -> private extensions (helpers,...) ) :)